wiki/en/Getting-help.md

```diff
@@ -0,0 +1,16 @@
+If you need help, ask your question in the [Gitter Chat](https://gitter.im/SpaceVim/SpaceVim), a member of the community will help you out.
+
+here is a simple guide for how to getting help about SpaceVim. Before asking question, please read the FAQ, the answer to your question maybe
+listed there. 
+
+
+### Transitioning from origin vim config
+
+If you are not new to Vim, and you have familial with your own vim config, We recommend to use `vimcompatible` mode, In this mode, no origin key
```

Comment:
* s/have familial/are familiar/
* s/We/we
* I _think_ you mean "no original key bindings will be overriden"